materials/03-Developing-gem5-models/05-modeling-cores/completed/processors.py

```diff
@@ -0,0 +1,152 @@
+# Copyright (c) 2022 The Regents of the University of California
```

Comment:
You've put these material files in the wrong directory. These should be in in "02-Using-gem5". This is "03-Developing-gem5-models". At present, on the main branch, these files are mixed in with those i use in my "adding an instruction" example.